.github/workflows/explorer.yml

```diff
@@ -0,0 +1,25 @@
+name: Explorer-CI
+
+on:
+  push:
+    paths:
+      - 'explorer/**'
+
+jobs:
+  Explorer:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Clone Scalardl-tool repository
+      uses: actions/checkout@v2
+
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: Build Explorer
```

Comment:
Please see comments in another PR.

Reply:
Thank you for review comment.
Workflow modified based on review comments.